components/member-profile/contribution/index.js

```diff
@@ -41,8 +41,14 @@ const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
     }
   }
 
+  let url = featureUrl || prList[0]?.url;
+
   return (
-    <div>
+    <div
+      className={url && classNames.contributionCard}
+      onClick={() => url && window.open(url, '_blank')}
+      onKeyDown={() => {}}
```

Comment:
why is it blank? Don't you think we should be using `onKeyPress` and do the same thing as `onClick`?

Reply:
its the eslint requirement to put the onKeyDown
so i made it empty
yes I should put the same thing changing it